### PR TITLE
Translate the basic strings file to French

### LINKED ISSUE #224: Translate the basic strings file to French
Steps:

1. Clone this repo
2. Copy the `package.nls.json` to `package.nls.fr.json`
3. Edit the file to replace all the English strings to French
4. Submit a PR

### PR DESCRIPTION
**Pull request by oleksis**
Fixes #224

### CODE REVIEW

File: package.nls.fr.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "vscodePets.nameYourPet": "Nommez votre animal", 
+    "vscodePets.cancelledSpawningPet":"Générer un animal a été annulé",
+    "vscodePets.spawnPetWithClosedPlayground": "Une place de jeux pour animaux a été créée. Vous pouvez désormais utiliser la Commande \"Spawn Additional Pet\" pour ajouter d'autres animaux.",
+    "vscodePets.removePetsWithClosedPlayground": "Une place de jeux pour animaux a été créée. Vous pouvez désormais utiliser la Commande \"Remove All Pets\" pour supprimer tous les animaux.",
+
+    "vscodePets.black": "Noir",
+    "vscodePets.brown": "Brun",
+    "vscodePets.green": "Vert",
+    "vscodePets.yellow": "Jaune",
+    "vscodePets.gray": "Gris ",
```
Comment (tonybaloney) [inline package.nls.fr.json:17]: Please remove the trailing space inside the quote

**Review by tonybaloney — CHANGES_REQUESTED** [review]

**Comment by tonybaloney** [comment]
Thanks for the contribution! I'm impressed you speak so many languages

File: package.nls.fr.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "vscodePets.spawnPet": "Générer un animal",
+    "vscodePets.petPanel": "Panneau pour animaux",
+    "vscodePets.selectPet": "Sélectionnez un animal",
+    "vscodePets.selectColor": "Sélectionnez un couleur",
```
Comment (StephenSorriaux) [inline package.nls.fr.json:5]: *une couleur

File: package.nls.fr.json (hunk 1 above)
Comment (oleksis) [reply package.nls.fr.json]: Typo fixed

**Review by tonybaloney — APPROVED** [review]

**Pull request merged**